packages/office-ui-fabric-react/src/components/Label/Label.visualtest.ts

```diff
@@ -0,0 +1,34 @@
+import { Casper, IPhantomCSS } from '../../visualtest/PhantomCssInterface';
+import { baseUrl } from '../../common/VisualTest';
+declare var phantomcss: IPhantomCSS;
+declare var casper: Casper;
+/* tslint:disable:no-function-expression */
+casper.
+  start(baseUrl + 'label').
+  then(function () {
+    phantomcss.screenshot('.Label', 'Label_not_pressed');
+  }).then(function () {
+    this.mouse.move('.Label');
+    phantomcss.screenshot('.Label', 'Label_hovered');
+  }).then(function () {
+    this.mouse.down('.Label');
```

Comment:
Label doesn't have hover/pressed states. It's only text. Let's remove these tests unless we're expecting something different.